components/Chips/src/MDCChipCollectionViewCell.m

```diff
@@ -71,7 +74,11 @@ - (CGSize)intrinsicContentSize {
 }
 
 - (void)setSelected:(BOOL)selected {
-  [super setSelected:selected];
+  // Skipping state management in the current cell. States is managed in self.chipView instead.
+  if (!self.chipView.enableRippleBehavior) {
+    [super setSelected:selected];
+  }
```

Comment:
This pattern is a bit concerning to me; we don't know what UIKit's doing in the super class, so not calling super may lead to surprising behavior in the future. Is it possible to avoid conditionally calling super?

Reply:
I'm also concerned that this may affect how UIAccessibility will work. I believe UIAccessibility automatically assigns the `.selected` trait to the element. If we're no longer calling through to the collection view cell's implementation, we'll have to reproduce all of the behavior locally.  Please perform thorough regression tests using both Voice Over and Switch Control to confirm that there is no change in behavior aside from the desired visual changes.